app/forms/webauthn_setup_form.rb

```diff
@@ -161,6 +160,12 @@ def mfa_user
     @mfa_user ||= MfaContext.new(user)
   end
 
+  def aaguid
+    attestation_response&.authenticator_data&.aaguid
+  rescue WebAuthn::AuthenticatorDataFormatError
+     nil
```

Comment:
Ok looks like we are getting `StandardError` after all? 🤷 

From adding a `binding.pry` and running failing spec `rspec ./spec/forms/webauthn_setup_form_spec.rb:226`.

```
From: /Users/andrewmduthie/Documents/Code/identity-idp/app/forms/webauthn_setup_form.rb:167 WebauthnSetupForm#aaguid:

    163: def aaguid
    164:   attestation_response&.authenticator_data&.aaguid
    165: rescue => e
    166:   binding.pry
 => 167:   nil
    168: end

[1] pry(#<WebauthnSetupForm>)> e
=> #<StandardError: StandardError>
```

Curious where this is being raised...

```suggestion
  rescue StandardError
    nil
```

Reply:
Oh.... it's because it's what we stub 😕 

https://github.com/18F/identity-idp/blob/f982fda6b55cbb2e58534b839ade1d6c52802250/spec/forms/webauthn_setup_form_spec.rb#L222-L224

I'm not sure that stub accurately reflects reality.

Reply:
Yea I think we can change it to reflect webauthn data error

Reply:
But it doesnt explain why the other tests are failing as well.